.dockerignore

```diff
@@ -0,0 +1,7 @@
+.circleci
+.nyc_output
+.repos
+coverage
+lib
+node_modules
+types
```

Comment:
Given that the tests aren't run in the `compile` container and `test/` doesn't end in the final container, I'd also include it here.

Reply:
Took me a minute to understand what you mean, but you are right! Sweet catch! 👍